tests/src/whisk/core/entity/test/DatastoreTests.scala

```diff
@@ -77,16 +77,6 @@ class DatastoreTests extends FlatSpec
         putGetCheck(authstore, auth, WhiskAuth)
     }
 
-    it should "lookup key by uuid" in {
```

Comment:
I don't think so, functionality is gone.

We need testcoverage for the `Identity` object/class though.